src/pages/landing/components/RegisterModal.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
 
 const RegisterModal = (props: Props) => {
   return (
-    <Dialog open={props.show} onClose={props.onClose}>
+    <Dialog open={props.show} onClose={props.onClose} role="dialog">
```

Comment:
Adding role attribute was great, for accessibility. But for using in tests, could you introduce data-testid instead